.github/workflows/release.yml

```diff
@@ -125,7 +125,7 @@ jobs:
     container: quay.io/pypa/manylinux_2_28_x86_64
     strategy:
       matrix:
-        python-version: ["cp3.9", "cp3.10", "cp3.11", "pp3.9"]
+        python-version: ["cp3.10", "cp3.11", "pp3.9"]
```

Comment:
why no 3.12 here?

Reply:
you're right here and below, just an oversight that I left them out.

Reply:
We do need 3.12 in the release script but this isn't the right PR for that.